x-pack/solutions/search/plugins/search_indices/public/plugin.ts

```diff
@@ -88,6 +89,22 @@ export class SearchIndicesPlugin
       },
       visibleIn: [],
     });
+    core.application.register({
+      id: SEARCH_INDEX_MANAGEMENT_APP_ID,
+      appRoute: SEARCH_INDEX_MANAGEMENT_APP_BASE,
+      title: i18n.translate('xpack.searchIndices.elasticsearchIndices.indexManagementTitle', {
+        defaultMessage: 'Index Management',
+      }),
+      async mount({ element, history }) {
+        const { renderIndexManagementApp } = await import('./index_management_application');
```

Comment:
I'm pretty sure this should be able to directly use the IndexManagement.indexManagementApp

Reply:
I don't think thats the case. We started with just rendering the index management app returned by `mountManagementSection` and need to wrap it in a `PageTemplate` and add a section to get the right formatting.